.gitattributes

```diff
@@ -0,0 +1,7 @@
+/.coveralls.yml export-ignore
+/.gitattributes export-ignore
+/.gitignore export-ignore
+/.travis.yml export-ignore
+/build.xml
+/phpunit.xml.dist
```

Comment:
Hi @thePanz and thank you for your PR. 

As I'm not experienced in gitattributes, I wonder what the two "path without attributes" lines will result in?

Reply:
Sorry, the `export-ignore` parts are required, added!